spec/models/miq_report/search_spec.rb

```diff
@@ -88,4 +116,13 @@
       expect(results.length).to eq(20)
     end
   end
+
+  private
+
+  # stock to_sql is not supported by all arel nodes (noteably: attribute)
+  # So this is needed to test the generated sql
+  def to_s(nodes)
+    @visitor = Arel::Visitors::ToSql.new MiqReport.connection
+    Array.wrap(nodes).map { |node| @visitor.accept(node, Arel::Collectors::SQLString.new).value }
+  end
```

Comment:
/cc @imtayadeway the `to_sql` method doesn't work so well. Very quirky.
You may want to use something like this, or maybe we should pull out into a global helper e.g.: `arel_to_sql`